packages/common/src/users/_internal/UserUiSchemaSettings.ts

```diff
@@ -39,6 +82,66 @@ export const buildUiSchema = async (
           },
         ],
       },
+      {
+        type: "Section",
+        labelKey: `${i18nScope}.sections.orgSettings`,
+        options: {
+          helperText: {
+            label: `${context.portal.name}`,
+          },
+        },
+        rules: [
+          {
+            effect: UiSchemaRuleEffects.SHOW,
+            conditions: [
+              context.isOrgAdmin &&
+                context.currentUser.orgId === context.portal.id,
+            ],
+          },
+        ],
+        elements: [
+          {
+            type: "Notice",
+            options: {
+              notice: {
+                configuration: {
+                  id: "user-org-settings-notice",
+                  noticeType: "notice",
+                  closable: false,
+                  kind: "info",
+                  scale: "m",
+                },
+                title: `{{${i18nScope}.notice.title:translate}}`,
+                message: noticeMessage,
+                autoShow: true,
+                actions: orgNoticeActions,
+              },
+            },
+          },
+        ],
+      },
     ],
   };
 };
+
+async function _getCommunityOrEnterpriseUrl(
```

Comment:
Two thoughts/suggestions: 
1. Maybe call this `getCommunityOrEnterpriseAGOUrl`?
2. Just wondering if this should be abstracted out into a reusable util?